app/workers/mute_worker.rb

```diff
@@ -2,10 +2,26 @@
 
 class MuteWorker
   include Sidekiq::Worker
+  include Redisable
 
   def perform(account_id, target_account_id)
-    FeedManager.instance.clear_from_home(Account.find(account_id), Account.find(target_account_id))
+    @account        = Account.find(account_id)
+    @target_account = Account.find(target_account_id)
```

Comment:
Tried to change this to follow the pattern used in the services, but I'm not sure if it's ideal.